webv2/web.go

```diff
@@ -2279,7 +2289,7 @@ func getGeneratedResources(w http.ResponseWriter, r *http.Request) {
 	}
 	if sessionState.Conv.Audit.StreamingStats.DataflowJobId != "" {
 		generatedResources.DataflowJobName = sessionState.Conv.Audit.StreamingStats.DataflowJobId
-		generatedResources.DataflowJobUrl = fmt.Sprintf("https://console.cloud.google.com/dataflow/jobs/%v/%v?project=%v", sessionState.Region, sessionState.Conv.Audit.StreamingStats.DataflowJobId, sessionState.GCPProjectID)
+		generatedResources.DataflowJobUrl = fmt.Sprintf("https://console.cloud.google.com/dataflow/jobs/%v/%v?project=%v", sessionState.Conv.Audit.StreamingStats.DataflowLocation, sessionState.Conv.Audit.StreamingStats.DataflowJobId, sessionState.GCPProjectID)
```

Comment:
We should also store `region` separately inside the generatedResources for Dataflow in addition to the URL. Similarly for other configurations such as the `hostProjectId`. 

 Once we start persisting generatedResources in the metadata database, we will need these values individually to perform operations on them - and interpreting the URL to get these values will be prone to error.

Reply:
We should take this refactoring up separately along with other resources like pubsub datastream etc. Filed b/311343651 to track.